### PR TITLE
Upgrade to Fizzy to 0.3.0

### LINKED ISSUE #141: Use fizzy 0.3.0

### PR DESCRIPTION
**Pull request by axic**
Fixes #141.

### CODE REVIEW

**Comment by jwasinger** [comment]
Testing this

**Comment by axic** [comment]
It is not released yet :)

**Comment by axic** [comment]
Should merge #126 first.

**Comment by axic** [comment]
@jwasinger well, it is fine to test this now, it has been released :)

File: scout-engines/Dockerfile [hunk 1]
```diff
@@ -187,7 +187,7 @@ RUN cd eip1962-bls12-rs-native && cargo build --release
 WORKDIR /engines
 
 # fizzy for bls12 using host funcs
-RUN git clone --single-branch --branch ewasm-bench-bls12 https://github.com/wasmx/fizzy.git fizzy-bls12-hostfuncs && \
+RUN git clone --single-branch --branch v0.3.0-ewasm-bench-bls12 https://github.com/ewasm-benchmarking/fizzy.git fizzy-bls12-hostfuncs && \
```
Comment (jwasinger) [inline scout-engines/Dockerfile:190]: this branch doesn't build:

```
                                                                                                                       
/engines/fizzy-bls12-hostfuncs/test/utils/fizzy_engine.cpp:75:8: error: no type named 'execution_result' in namespace 'fizzy'; did you mean 'ExecutionResult'?                                             
fizzy::execution_result bignum_int_add(                                                                                                                                                                    
~~~~~~~^~~~~~~~~~~~~~~~                                                                                                                                                                                    
       ExecutionResult                                                                                                                                                                                     
/engines/fizzy-bls12-hostfuncs/lib/fizzy/execute.hpp:18:8: note: 'ExecutionResult' declared here                                                                                                           
struct ExecutionResult                                                                                                                                                                                     
       ^                                                                                                                                                                                                   
/engines/fizzy-bls12-hostfuncs/test/utils/fizzy_engine.cpp:85:12: error: no matching constructor for initialization of 'fizzy::ExecutionResult'                                                            
    return {false, {ret}};                                                                                                                                                                                 
           ^~~~~~~~~~~~~~                                                                                                                                                                                  
/engines/fizzy-bls12-hostfuncs/lib/fizzy/execute.hpp:25:15: note: candidate constructor not viable: requires single argument '_value', but 2 arguments were provided                                       
    constexpr ExecutionResult(uint64_t _value) noexcept : has_value{true}, value{_value} {}                                                                                                                
              ^                                                                                                                                                                                            
/engines/fizzy-bls12-hostfuncs/lib/fizzy/execute.hpp:29:24: note: candidate constructor not viable: requires single argument 'success', but 2 arguments were provided                                      
    constexpr explicit ExecutionResult(bool success) noexcept : trapped{!success} {}                                                                                                                       
                       ^                                                                                                                                                                                   
/engines/fizzy-bls12-hostfuncs/lib/fizzy/execute.hpp:18:8: note: candidate constructor (the implicit copy constructor) not viable: requires 1 argument, but 2 were provided                                
struct ExecutionResult                                                                                                                                                                                     
       ^
...                                                                                                                           
```

Comment (axic) [reply scout-engines/Dockerfile:190]: It does as of 3 hrs ago.

**Pull request merged**